test/fixtures/video-players.html

```diff
@@ -8,7 +8,7 @@
   <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
 
   <!-- (1) Include your video player component here -->
-  <script async custom-element="amp-youtube" src="/dist/v0/amp-youtube-0.1.max.js"></script>
+  <script async custom-element="amp-ooyala-player" src="/dist/v0/amp-ooyala-player-0.1.max.js"></script>
```

Comment:
This needs to have its own line instead of replacing youtube tests. so we need
```
<script async custom-element="amp-youtube" src="/dist/v0/amp-youtube-0.1.max.js"></script>
<script async custom-element="amp-ooyala-player" src="/dist/v0/amp-ooyala-player-0.1.max.js"></script>
```